dbcon/joblist/crossenginestep.cpp

```diff
@@ -525,17 +545,16 @@ void CrossEngineStep::execute()
                 }
 
                 addRow(rgDataDelivered);
+
+                if (rowFe1.usesStringTable() && fe1RGData.getStringTableMemUsage() > 50 * (1 << 20))
+                {
+                    fe1RGData.clearStringStore();
+                }
```

Comment:
We should move this threshold value (50 * (1 << 20)) into a common location (maybe in rowgroup.h) as a const int or a macro, so it can be easily updated later.

Use of UNLIKELY around the if predicate might be a suitable optimization?